test_ir.py

```diff
@@ -18,5 +18,14 @@ class IRkernelTests(jkt.KernelTests):
     complete_code_samples = ['1', "print('hello, world')", "f <- function(x) {\n  x*2\n}"]
     incomplete_code_samples = ["print('hello", "f <- function(x) {\n  x*2"]
 
+class InstallspecTests(jkt.KernelTests):
+    # just a small test, it's the same kernel after all...
+    kernel_name = "testir"
+
+    language_name = "R"
+
+    code_hello_world = "print('hello, world')"
+
+
```

Comment:
good incentive to start being consistent in the first place :stuck_out_tongue_winking_eye: 

and in this case we only have like 20 lines in there yet anyway.

also please remember when i re-did *all* the IRkernel code style. i’m sure you like working on the codebase more now that it doesn’t look like it did before.